api-client/build.gradle.kts

```diff
@@ -5,6 +5,9 @@ plugins {
 android {
     namespace = "com.theolm.temp"
 }
+dependencies {
```

Comment:
Do we need this dependency here?

Reply:
It's removal seems to have no impact. So i'll take it out